Distiller-server/src/main/java/com/mapr/distiller/server/recordtypes/SystemCpuRecord.java

```diff
@@ -73,6 +77,53 @@ public SystemCpuRecord() throws Exception {
 					this.cpu_softirq.add(
 					this.cpu_steal.add(
 					this.cpu_other))))))));
+			
+			//Now process single cpu lines
+			LinkedList<String[]> cpuLines = new LinkedList<String[]>();
```

Comment:
I think we should ArrayList instead of LinkedList. LinkedList takes O(n) traversals for a get() rather ArrayList (backed by Array) is O(1) for a get().